### ISSUE TITLE
A way to lose Aggro

### ISSUE THREAD

**Issue by Kamogli** [Marsbars/DungeonCrawler-Issues#14]
Howdy Mars,

one thing I noticed and was thinking about: How to go about aggro gains as a ranged DD or Healer? Currently, it'll fight the mobs, standing. It's probably messy but would it be possible to walk the distance to the tank once aggro is gained, for him to lose the aggro for you?

Don't know if it makes sense ... 

Cheers,

**Comment by Marsbars** [comment]
Yeah I see what you mean and it does make sense, I'll see what I can put together.
